lib/ansible/modules/remote_management/hpilo/hponcfg.py

```diff
@@ -40,6 +40,20 @@
   minfw:
     description:
     - The minimum firmware level needed
+    required: false
+  hponcfg:
+    description:
+    - Path to the hponcfg executable (`hponcfg` which uses $PATH)
```

Comment:
Descriptions should end with a dot. (Yes, the original module is doing this wrong as well because we weren't that strict in the olden days :-p)